### ISSUE TITLE
Component name "Tasks" should always be multi-word  vue/multi-word-component-names

### ISSUE THREAD

**Issue by akakiev** [Rose-stack/building-a-vue-app-with-django-rest-framework#1]
at - django_rest_api\django_rest_api_tasks\client\django_tasks_client\src\components\Tasks.vue

get - 1:1  error  Component name "Tasks" should always be multi-word  vue/multi-word-component-names

and it is not work!

**Comment by andrealcantara** [comment]
it's the same for me.

**Comment by Mute-yc** [comment]
I meet the same trouble, did you find a solution?

**Comment by akakiev** [comment]
no, still don't find...I hope that the author will answer here, but NO...

**Comment by Rose-stack** [comment]
@andrealcantara @akakiev @Mute-yc 

Which version of Vue are you using to create your Vue app?

I used Vue 2

**Comment by Rose-stack** [comment]
@andrealcantara @akakiev @Mute-yc

If you are using Vue 3

Check the main.js file and refactor that based on Vue 3 instance of the app

**Comment by Rose-stack** [comment]
https://plainenglish.io/blog/differences-between-vue-2-and-vue-3
